modules/services/window-managers/i3-sway/lib/functions.nix

```diff
@@ -39,18 +39,22 @@ rec {
     concatStringsSep "\n"
     (map (c: "assign ${criteriaStr c} ${workspace}") criteria);
 
-  fontConfigStr = fontNames: fontSize:
-    optionalString (fontNames != [ ])
-    "font pango:${concatStringsSep ", " fontNames} ${toString fontSize}";
+  fontConfigStr = { names, style ? "", size }:
```

Comment:
nitpick: the default `""` for `style` isn't needed if the corresponding option already has that default value, I think?